api/controllers.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
     return db.getProfile(req.query.userId)
       .then((response) => {
         if (!response || response.rowCount === 0) {
-          res.status(400).send(`Unable to get user profile based on userId: ${req.query.userId}`);
+          res.status(400).send('Unable to get user profile based on userId');
```

Comment:
surprises me that this was a security issue

Reply:
Yeah, SonarCloud gave me lots of links explaining how it is a huge XSS vulnerability + ways to refactor. It pointed me to sanitizeHTML npm module, but I think this refactor was faster/easier